com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,25 @@
+app-id: com.github.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
+rename-desktop-file: tuxpusher.desktop
+rename-appdata-file: tuxpusher.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - name: tuxpusher
+    buildsystem: simple
+    build-commands:
+      - cd TuxPusher_SDL;make;make install
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.desktop ${FLATPAK_DEST}/share/applications/tuxpusher.desktop
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.appdata.xml ${FLATPAK_DEST}/share/metainfo/tuxpusher.appdata.xml
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.png ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/tuxpusher.png
```

Comment:
.png does not belong into scalable

Reply:
Corrected this to `${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/tuxpusher.png` is that suitable?